clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/op/job/KubectlJobExecutor.java

```diff
@@ -895,7 +895,7 @@ private Retry getRetry(RetryRegistry retryRegistry, String identifier) {
       // Hence, we need to translate that exception back into a JobResult object - but
       // we only need to do it for KubectlException (since that is explicitly thrown above) and not
       // for any other ones.
-      return finalResult.build();
+      return finalResult.error(e.getMessage()).build();
```

Comment:
Doesn't finalResult.error get set above?

Reply:
Ohh that's right. I misread the logs and thought the actual error was not being printed.